Kickstarter-iOS/Views/LiveStreamNavTitleView.swift

```diff
@@ -0,0 +1,96 @@
+import KsApi
+import Library
+import LiveStream
+import Prelude
+import ReactiveSwift
+import UIKit
+
+internal protocol LiveStreamNavTitleViewDelegate: class {
+  func liveStreamNavTitleView(_ navTitleView: LiveStreamNavTitleView,
+                              requiresLayoutWithPreferredSize size: CGSize)
+}
+
+internal final class LiveStreamNavTitleView: UIView {
+
+  @IBOutlet private weak var rootStackView: UIStackView!
+  @IBOutlet private weak var playbackStateLabel: UILabel!
+  @IBOutlet private weak var playbackStateLabelContainer: UIView!
+  @IBOutlet private weak var eyeImageView: UIImageView!
+  @IBOutlet private weak var numberOfPeopleWatchingLabel: UILabel!
+  @IBOutlet private weak var numberOfPeopleWatchingLabelContainer: UIView!
+  @IBOutlet private weak var numberOfPeopleWatchingStackView: UIStackView!
+
+  private weak var delegate: LiveStreamNavTitleViewDelegate?
+  private let viewModel: LiveStreamNavTitleViewModelType = LiveStreamNavTitleViewModel()
+
+  internal class func fromNib() -> LiveStreamNavTitleView {
+    return UINib(nibName: Nib.LiveStreamNavTitleView.rawValue, bundle: .framework)
+      .instantiate(withOwner: nil, options: nil)
+      //swiftlint:disable:next force_cast
+      .first as! LiveStreamNavTitleView
```

Comment:
Think we should assert here if this fails, it's kind've similar to our `configureCell` not finding a registered cell type?